cli/args_test.go

```diff
@@ -163,7 +169,7 @@ func assertOptionsEqual(t *testing.T, expected options.TerragruntOptions, actual
 	assert.Equal(t, expected.IamRole, actual.IamRole, msgAndArgs...)
 }
 
-func mockOptions(t *testing.T, terragruntConfigPath string, workingDir string, terraformCliArgs []string, nonInteractive bool, terragruntSource string, ignoreDependencyErrors bool, includeExternalDependencies bool) *options.TerragruntOptions {
+func mockOptions(t *testing.T, terragruntConfigPath string, workingDir string, terraformCliArgs []string, nonInteractive bool, terragruntSource string, ignoreDependencyErrors bool, ignoreDependencyOrder bool, includeExternalDependencies bool) *options.TerragruntOptions {
```

Comment:
Should this method use this variable?

Reply:
Not too sure to be honest ^_^ that seemed like the correct way to get that option tested.

Reply:
Hm, right now, it's a bit deceiving, as anyone using the `mockOptions`, `mockOptionsWithIamRole` and `mockCmdOptions` methods would expect that boolean to do something, but it's actually not set on the `TerragruntOptions` struct, so it does nothing. Since it's not being used in any tests now (always set to `false`), I'd remove this new param from these methods (they have lots of params already!) and we can bring it back in the future if we have a use case that merits overriding it.

Reply:
Hmmm, ok, but then how can I add a test that tests the `--terragrunt-ignore-dependency-errors` flag use? If I add 
```

Reply:
Probably the better bet would be an integration test in `integration_test.go` that sets the flag and makes sure things work as expected.

Reply:
Hmm, had a look at `integration_test.go` and it’s a bit tricky… Would you say such an integration test is necessary, or it’s good enough without?
(as in, happy to look deep into it if it’s a blocker :) )

Reply:
Fairy enough, can prob live without it. I'll kick off tests now.